dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -748,6 +754,7 @@ CONFIG_KVM_INTEL=y
 # CONFIG_KVM_INTEL_PROVE_VE is not set
 CONFIG_X86_SGX_KVM=y
 CONFIG_KVM_AMD=y
+# CONFIG_KVM_IOAPIC is not set
```

Comment:
@ramosian-glider do we want this one enabled? Seems to be new in 6.17.

Reply:
As discussed offline, we'd better enable it, as it is Y by default.

Reply:
Done.